src/m365/spo/commands/contenttype/contenttype-get.ts

```diff
@@ -43,7 +54,12 @@ class SpoContentTypeGetCommand extends SpoCommand {
       .get(requestOptions)
       .then((res: any): void => {
         if (res['odata.null'] === true) {
-          cb(new CommandError(`Content type with ID ${args.options.id} not found`));
+          if(args.options.id){
+            cb(new CommandError(`Content type with ID ${args.options.id} not found`));
+          }
+          if(args.options.contenttypeTitle){
```

Comment:
This case seems to be never hit. If you specify name of a content type, SharePoint returns `{ value: [] }` rather than `{ "odata.null": true }`, so when retrieving content types by name, we should check for content type based on the length of the `value` array.

Reply:
thanks for the detail check.. corrected.. when the res is based on request based on name option I do `res = res.value` to make the same response type